fendermint/app/src/cmd/objects.rs

```diff
@@ -72,23 +72,27 @@ cmd! {
                 }
 
                 let client = FendermintClient::new_http(tendermint_url, None)?;
-                let iroh_manager = IrohManager::from_addr(Some(iroh_addr));
+                let iroh_addr = iroh_addr
+                    .to_socket_addrs()?
```

Comment:
Makes sense. Why did we start using Iroh Manager earlier? Does it provide connection pooling?

Reply:
it provides lazy connection creation... it's something we wrote... it's getting removed with the iroh upgrade work